src/components/Clothing.spec.js

```diff
@@ -26,12 +34,12 @@ describe('Clothing', () => {
     expect(url).toBeInTheDocument()
   })
 
-  it('has a button', () => {
+  it('has two buttons', () => {
     render(<Clothing imageUrl={imageUrl} />)
 
-    const imageButton = screen.getByRole('button')
+    const imageButton = screen.getAllByRole('button')
 
-    expect(imageButton).toBeInTheDocument()
+    expect(imageButton).toHaveLength(2)
   })
 
   it('it does not render any html elements', () => {
```

Comment:
you can drop the "it" in the description of the test :)
```suggestion
  it('does not render any html elements', () => {
```